lib/SILOptimizer/SILCombiner/SILCombinerMiscVisitors.cpp

```diff
@@ -1302,6 +1334,9 @@ visitUnreachableInst(UnreachableInst *UI) {
 SILInstruction *
 SILCombiner::
 visitUncheckedTakeEnumDataAddrInst(UncheckedTakeEnumDataAddrInst *TEDAI) {
+  if (TEDAI->getFunction()->hasOwnership())
```

Comment:
Can you split out the hasOwnership move into its own commit? That way in this commit, you can show that you are removing the partial apply one? Then it is clear what the code motion is.

Reply:
I see that you actual did split it into two commits. Can you improve the commit message? Say what you are doing.

Reply:
@gottesmm I've updated the commit messages both here and in #30559. 
d3ba5fe isn't changed here but that's OK because it will go away after #30559 lands and I rebase.